### PR TITLE
feat: Add project names above images + put in actual slides

### LINKED ISSUE #32: 🚀 Feature: Update with real slides

### PR DESCRIPTION
**Pull request by DonLe27**
Resolves #32

### CODE REVIEW

File: server/src/index.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 const http = require('http');
+const dotenv = require('dotenv');
```
Comment (BryanPan342) [inline server/src/index.js]: remove this change

**Review by BryanPan342 — CHANGES_REQUESTED** [review]
@DonLe27 remove changes to server/src/index.js and remove the test.glb file

File: server/src/index.js (hunk 1 above)
Comment (DonLe27) [reply server/src/index.js]: done thanks

**Review by arjunr00 — DISMISSED** [review]
lgtm

**Review by josephtclegg — COMMENTED** [review]
looks like we're off to the races

**Review by BryanPan342 — APPROVED** [review]

**Pull request merged**